### PR TITLE
Edinburgh Hallucination Leaderboard

### PR DESCRIPTION
**Pull request by clefourrier**
Reviewable now! :)

### CODE REVIEW

**Comment by clefourrier** [comment]
Needs to be proofread by Pasquale Minervi first

File: leaderboards-on-the-hub-hallucinations.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+- user: clefourrier
+- user: rohitsaxena
+  guest: true
+- user: aryopg
+  guest: true
+- user: zodiache
+  guest: true
+---
+
+
```
[the remark this answers is not in the document]
Comment (clefourrier) [reply leaderboards-on-the-hub-hallucinations.md:18]: I initially suggested putting the University of Edinburgh in the title but @pminervini was not sure their name can be used like that, so we went for a more generic title

[the remark this answers is not in the document]
Comment (thomwolf) [reply leaderboards-on-the-hub-hallucinations.md:19]: I think it's fine as it's a guest post

File: leaderboards-on-the-hub-hallucinations.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+## A glance at the results so far
+
+We are currently in the process of evaluating a very large number of models from the Hugging Face Hub – we can analyse some of the preliminary results. For example, we can draw a clustered heatmap resulting from hierarchical clustering of the rows (datasets and metrics) and columns (models) of the results matrix. 
+
+![Clustermap All](https://huggingface.co/spaces/hallucinations-leaderboard/leaderboard/resolve/main/blog/figures/clustermap_all_viridis.png)
+
+We can identify the following clusters among models:
+Mistral 7B-based models (Mistral 7B-OpenOrca, zephyr 7B beta, Starling-LM 7B alpha, Mistral 7B Instruct, etc.)
+LLaMA 2-based models (LLaMA2 7B, LLaMA2 7B Chat, LLaMA2 13B, Wizard Vicuna 13B, etc.)
+Mostly smaller models (BLOOM 560M, GPT-Neo 125m, GPT-Neo 2.7B, Orca Mini 3B, etc.)
```
Comment (pcuenca) [inline leaderboards-on-the-hub-hallucinations.md:94]:
```suggestion
- Mistral 7B-based models (Mistral 7B-OpenOrca, zephyr 7B beta, Starling-LM 7B alpha, Mistral 7B Instruct, etc.)
- LLaMA 2-based models (LLaMA2 7B, LLaMA2 7B Chat, LLaMA2 13B, Wizard Vicuna 13B, etc.)
- Smaller models like BLOOM 560M, GPT-Neo 125m, GPT-Neo 2.7B, Orca Mini 3B, etc.
```

**Review by thomwolf — APPROVED** [review]
all good to me

**Review by pcuenca — APPROVED** [review]
Looks good! As a potential improvement, I think a short discussion on benchmark limitations could be useful to interested readers not familiar with the topic. I may be mistaken, but it looks like some of the comments in the results analysis imply that high-scoring in some benchmarks does not necessarily correlate with great real-world performance.

**Pull request merged**